src/analysis/retail/priest/shadow/modules/guide/CooldownGraphSubsection.tsx

```diff
@@ -80,6 +82,38 @@ const CoreCooldownsGraph = () => {
     );
   }
 
+  /* There are some issues with SW:D highlight sections that need to be worked out furtuer before implementation.
```

Comment:
i will investigate this

Reply:
Yea that makes sense. I saw you added some code to merge overlapping ranges. The crash is caused when *no* enemies have an execute range for SW:D. I have a fix for it inbound.

Reply:
Thank you! Sorry I didn't finish it myself, when I saw that the new shadow priest talents were causing trouble I tried to get something up and running as soon as I could.

Reply:
all good! shadow got a ton of changes, i'm impressed you got this out so fast!